_posts/2017/2017-03-14-one-development-environment-to-rule-them-all.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "One development environment to rule them all"
+date: "2017-03-14 07:00:00"
+image: blog-header/ring.jpg
+categories: tech-stories
+authors: ["Jordi"]
+---
+
+A usual problem developers are facing, is to work with a common development environment.
+At the same time, this environment has to be the same as the production environment.
+ePages software is working into servers with centOS or debian which makes it mandatory for the developers to write the code under this environment.
```

Comment:
Please add links:

[CentOS](https://www.centos.org/)
[Debian](https://www.debian.org/)

... write the code in this environment.